### PR TITLE
2018-05-24 WG #5 Notes

### PR DESCRIPTION
**Pull request by tinnou**

### CODE REVIEW

File: notes/2018-05-24.md [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+Thoughts:
+ - Error codes/ identifiers for rules (Rules have multiple components, so an id per rule wouldn’t be granular enough)
+ - Identifying substring?
+ - There should be a programmatic way to understand the test results
+ - We can add testing specification to CATS and later add it to the spec if needed
+
+Proposed solution: 
+Have CATS introduce some concept of a mapping, that maps from implementation specific errors to GraphQL spec errors.  Each implementation can contribute to CATS by adding the list of error messages for each spec rule component. OL will open a new issue on CATS to show how this would work + examples
+
+### Transport level batching (Executing multiple operations in a single request)
```
Comment (IvanGoncharov) [inline notes/2018-05-24.md]: Hi @tinnou,

Just quick follow up: Are you interested in standardizing how batch requests are sent over HTTP?
On one of the previous GraphQL WG we discussed "GraphQL over HTTP" spec ([Notes](https://github.com/graphql/graphql-wg/blob/master/notes/2018-02-01.md#next-steps-for-graphql-over-http-spec), [Slides](https://docs.google.com/presentation/d/1udSoWIyLXJulWjO8cdm4BihKGH_U7ifEe7ZBwOg16M4/edit?usp=sharing), [Draft](https://github.com/APIs-guru/graphql-over-http)) with a goal to standartise HTTP request/responce for single and batch requests.

Are you interested in collaborating on such specification?

**Comment by leebyron** [comment]
Thanks for taking detailed notes!

File: notes/2018-05-24.md (hunk 1 above)
Comment (tinnou) [reply notes/2018-05-24.md]: @IvanGoncharov, I would love to help. Let's connect to figure out expectations and the parts I can contribute to.

Comment (IvanGoncharov) [reply notes/2018-05-24.md]: @tinnou Would be great 🎉 GitHub doesn't show your email so can you please write something on `ivan.goncharov.ua@gmail.com`?

Comment (IvanGoncharov) [reply notes/2018-05-24.md]: @tinnou Got your email. Thanks 👍

**Pull request merged**